src/components/FlightList.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { AIRLINES_LOGOS } from "../Constant";
+import {
+  formatDateTime,
+  generateRandomNumber,
+  getStatusColor,
+} from "../utility";
+
+const FlightList = ({ flightData = [], index = 0, handleFlightClick }) => {
+  return (
+    <>
+      <div
+        className="data-container"
+        style={{ marginTop: index === 0 && "50px" }}
```

Comment:
`marginTop: index === 0 ? "50px" :''`

always use ternary instead of `&&` for selection.